quixstreams/platforms/quix/topic_manager.py

```diff
@@ -55,12 +55,14 @@ def _create_topics(self, topics: List[Topic]):
 
     def _apply_topic_prefix(self, name: str) -> str:
         """
-        Prepend workspace ID to a given topic name
+        Prepend workspace ID to a given topic name, if required.
 
         :param name: topic name
 
         :return: name with workspace ID prepended
         """
```

Comment:
I think it's time to also rename this method to something like `_resolve_topic_name()` and explain what happens inside (Quix prefixing some topics).